lib/models/playing_cards.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+enum CardSuit {
+  spades,
+  hearts,
+  diamonds,
+  clubs,
+}
+
+enum CardType {
+  one,
+  two,
+  three,
+  four,
+  five,
+  six,
+  seven,
+  eight,
+  nine,
+  ten,
+  jack,
+  queen,
+  king
+}
+
+enum CardColor {
+  red,
+  black,
+}
+
+// Simple playing card model
+class PlayingCard {
+  CardSuit cardSuit;
+  CardType cardType;
+  bool faceUp;
```

Comment:
We are not doing that. Every card is face down(hidden) until is sent to a "graveyard" after a set is done.

Reply:
I'll try again and make changes accordingly tonight

Reply:
What do you think about #6 ?